homeassistant/components/flexpool/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Config flow for Flexpool integration."""
+import logging
+import re
+
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("address"): str,
```

Comment:
You can use the constant `homeassistant.const.CONF_ADDRESS` here to keep things standard.